src/webview/webview2/mod.rs

```diff
@@ -828,6 +859,35 @@ window.addEventListener('mousemove', (e) => window.chrome.webview.postMessage('_
     let _ = unsafe { self.webview.Navigate(PCWSTR::from_raw(url.as_ptr())) };
   }
 
+  pub fn load_url_with_headers(&self, url: &str, headers: http::HeaderMap) {
```

Comment:
we should try to reuse this function in the Webview::new code too, it's too big to just copy paste it.

Reply:
fixed now! I forgot to remove the duplicate code and the one in `Webview::new` was missing the `GET` method.